### DISCUSSION TITLE
Daily Standup (Wednesday, April 12th)

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#84] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on "add table" screen in Figma

**What are your priorities for the next 24 hours?**:
 - Finish that screen and others

**Blockers**:
 - None

**Shoutouts**:

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Start Figma
 - Organize our GitHub

**What are your priorities for the next 24 hours?**:
 - Finish dashboard screens with @Jhoscar22 

**Blockers**:
 - None atm

**Shoutouts**:
 - @Jhoscar22 for working on the figma

**Comment by Jhoscar22** [comment]
**What did you achieve in the last 24 hours?**:
 - Start Mockup, familiarize myself with Carbon Design

**What are your priorities for the next 24 hours?**:
 - Work as much as possible on Mockup

**Blockers**:
 - The dashboard has some complex components

**Shoutouts**:
 - @Int-Angel for his progress on the Figma
